### ISSUE TITLE
Other Commands/Custom Commands

### ISSUE THREAD

**Issue by Tanuj27** [dankgrinder/dankgrinder#88]
Add pls snakeeyes, pls slots, pls bet to get the badges
Use of pizza, horseshoe, daily
custom commands don't work as intended, cannot add interval, shows no error

**Comment by V4NSH4J** [comment]
"custom commands don't work as intended" 

What do you mean by this? 

"cannot add interval"

Why not? 

"shows no error"

Error for what?


If you don't have the answer to these questions, please use custom commands properly. They work exactly as intended.

**Comment by ghost** [comment]
yes you can add thos commands in custom command and read the Readme for more info 😀
example this also given in Readme 
custom_commands:
  - value: "pls se max"
    interval: 10
  - value: "pls bet max"
    interval: 12
    amount: 5
  - value: "pls slots max"
    Interval: 5

**Comment by Tanuj27** [comment]
nvm that, I figured it out.

Will you be adding for the new update anytime soon?

**Comment by V4NSH4J** [comment]
> nvm that, I figured it out. Will you be adding for the new update anytime soon?
.

I had finished working on it but I'm unsure if I am going to continue updating.

**Comment by Tanuj27** [comment]
Oh okay, that's cool.
